### PR TITLE
[CU-2w3b408] docs(frame): create oracle readme

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/2w3b408

## Description
Adds a readme for the oracle pallet.
Cross-reference pallets & integration and product content.
Run extrinsics scraper.
Line formatting for readability.

## Checklist
- [X] Style & expression.

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2w3b408 Oracle pallet readme](https://app.clickup.com/t/2w3b408)

File: frame/oracle/README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-# Apollo
```
Comment (vimukthi-git) [inline frame/oracle/README.md:1]: Are we no longer calling the oracle, Apollo?

Comment (Martin-Kiss) [reply frame/oracle/README.md:1]: That's a copy-paste error.

**Review by benluelo — APPROVED** [review]

File: frame/oracle/README.md [hunk 2]
```diff
@@ -1 +1,30 @@
 # Apollo
+
+The Oracle pallet provides functionality for setting up and maintaining an Apollo oracle and submitting prices.
```
Comment (vimukthi-git) [inline frame/oracle/README.md:3]: We can also link to the design doc for further details

**Review by vimukthi-git — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]
add design dock, and link to the mdbook + make sure it looks to the actual apollo portion of the repo

**Comment by Martin-Kiss** [comment]
Reference to the technical documentation now looks like the following in the product section:
image

The reference to the products section in pallets and integration:
image

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request merged**